stdlib/public/Platform/glibc.modulemap.in.gyb

```diff
@@ -37,28 +37,29 @@ module SwiftGlibc [system] {
       export *
     }
 
-    // glibc specific
+% if CMAKE_SDK != "FREEBSD":
     module features {
       header "@GLIBC_INCLUDE_PATH@/features.h"
       export *
     }
+% end
 
     module fenv {
       header "@GLIBC_INCLUDE_PATH@/fenv.h"
       export *
     }
-    
+
     // note: supplied by compiler
     // module float {
     //   header "@GLIBC_INCLUDE_PATH@/float.h"
```

Comment:
Ah, very true. Would you mind terribly if I did this in a separate pull request?